server/lib/dev_tool/fake_hydra/router.ex

```diff
@@ -4,8 +4,19 @@ defmodule DevTool.FakeHydra.Router do
   """
   use DevTool, :router
 
+  pipeline :browser do
+    plug(:accepts, ["html"])
+    plug(:fetch_session)
+    # plug :fetch_live_flash
+    plug(:put_root_layout, {DevTool.LayoutView, :root})
+    # plug :protect_from_forgery
+    plug(:put_secure_browser_headers)
+  end
+
   scope "/", DevTool.FakeHydra do
+    pipe_through([:browser])
     get("/oauth2/auth", OAuth2Controller, :auth)
+    get("/choose/user", OAuth2Controller, :user)
```

Comment:
You could rename this route to match the path to the template file which is "oauth2/choose-user"

```suggestion
    get("/oauth2/choose-user", OAuth2Controller, :user)
```

Reply:
I did want to use oauth2 prefix in the route in order to do not have conflict with hydra routes.